tool/tctl/common/auth_command.go

```diff
@@ -644,6 +644,20 @@ func (a *AuthCommand) generateUserKeys(clusterAPI auth.ClientI) error {
 		return trace.Wrap(err)
 	}
 	fmt.Printf("\nThe credentials have been written to %s\n", strings.Join(filesWritten, ", "))
+
+	expires, err := key.TeleportTLSCertValidBefore()
+	if err != nil {
+		logrus.Warnf("Failed to check TTL validity: %v.", trace.Wrap(err))
+		return nil
```

Comment:
A couple of suggestions here:

1. (nit) Alias the "logrus" import to "log"

2. Log using WithError, don't fail:

```suggestion
		logrus.WithError(err).Warn("Failed to check TTL validity")
		// err swallowed on purpose
```